in OS X when trying to compile this I get this error

```
/Users/rcatalano/webthree-umbrella/solidity/libsolidity/interface/CompilerStack.cpp:400:24: error: call to 'move' is ambiguous
                                m_errors.push_back(move(err));
                                                   ^~~~
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/type_traits:1622:1: note: candidate function
      [with _Tp = std::__1::shared_ptr<dev::solidity::Error> &]
move(_Tp&& __t) _NOEXCEPT
^
/usr/local/include/boost/move/utility_core.hpp:211:77: note: candidate function [with T = std::__1::shared_ptr<dev::solidity::Error> &]
         inline typename ::boost::move_detail::remove_reference<T>::type && move(T&& t) BOOST_NOEXCEPT
                                                                            ^
[ 40%] Building CXX object libsolidity/CMakeFiles/solidity.dir/interface/SourceReferenceFormatter.cpp.o
1 error generated.
make[2]: *** [libsolidity/CMakeFiles/solidity.dir/interface/CompilerStack.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [libsolidity/CMakeFiles/solidity.dir/all] Error 2
make: *** [all] Error 2
```

--- a/libsolidity/interface/CompilerStack.cpp
+++ b/libsolidity/interface/CompilerStack.cpp
@@ -397,7 +397,7 @@ StringMap CompilerStack::loadMissingSources(SourceUnit const& _ast, std::string 
 				*err <<
 					errinfo_sourceLocation(import->location()) <<
 					errinfo_comment("Source not found: " + errorMessage);
-				m_errors.push_back(move(err));
+				m_errors.push_back(std::move(err));
 				continue;
 			}
 			else

